stacker/util.py

```diff
@@ -531,10 +536,69 @@ def create_cache_directories(self):
 
     def get_package_sources(self):
         """Makes remote python packages available for local use."""
+        # Checkout S3 repositories specified in config
```

Comment:
Does the fact that s3 comes before git have implications?  What if someone defines an s3 package source after a git source, could they be potentially confused by the results?